koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -288,10 +288,10 @@ def _get_file_url(self, source_filename):
 
     def _download_files(self, source_filename, target_filename=None):
         if not target_filename:
-            if source_filename.endswith(".tar.gz"):
-                target_filename = "image.tar.gz"
-            else:
-                target_filename = "image.tar"
+            target_filename = "image.tar"
+            extension = self.osbs().get_compression_extension()
+            if extension:
+                target_filename = target_filename + extension
```

Comment:
Use string formatting here instead as this can easily fail if we got unicode from osbs and try to append it to str.